project/backend/docker-compose.infra.yml

```
@@ -0,0 +1,64 @@
+version: '3.8'
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.123.0
+    container_name: otel-collector
+    volumes:
+      - ./collector-config.yaml:/etc/otel-collector/config.yaml
+    command: [ "--config=/etc/otel-collector/config.yaml" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)